test/Installer/testmsi.ps1

```diff
@@ -2,7 +2,8 @@
 # Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 param(
-    [Parameter(Mandatory=$true)][string]$inputMsi
+    [string]$InputMsi,
```

Comment:
I switched it to follow the style in build.ps1 or dotnet-install.ps1. If you think it's important I'll add it back.